ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/masq-subnet/cucushift-installer-rehearse-vsphere-ipi-ovn-masq-subnet-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
```

Comment:
@jechen0648 
 you created one new job to consumed this new workflow to test.  eg https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml#L2466-L2472